lint-configs/pyproject.toml

```diff
@@ -1,8 +1,39 @@
-[tool.black]
+[tool.ruff]
 line-length = 145
-target-version = ['py39']
-verbose = true
+target-version = "py39"
 
-[tool.isort]
-line_length = 145
-profile = "black"
+[tool.ruff.lint]
+select = [
+    "E",
```

Comment:
We cam up with this config when working on asset mocker https://github.com/phantomcyber/soar-asset-mocker/blob/main/pyproject.toml#L56 there are a lot of good rules that IMHO should be activated

Reply:
Activated these here as well 👍